tests/ops/service/processors/post_processor_strategy/test_post_processor_strategy_filter.py

```diff
@@ -366,3 +368,534 @@ def test_nested_array_path():
             }
         }
     ]
+
+
+@mock.patch("fides.api.models.privacy_request.PrivacyRequest.get_raw_access_results")
+def test_filter_by_invalid_dataset_reference(mock_method):
+
+    config = FilterPostProcessorConfiguration(
+        field="customerId",
+        value={"dataset_reference": "<instance_fides_key>.customer"},
```

Comment:
Although this string works for our tests, we should replace `<instance_fides_key>` with an actual value. I don't want to give the impression that the placeholder will be replaced inside the `process` function. The `<instance_fides_key>` placeholder will be replaced by an actual value when the connector is instantiated.
```suggestion
        value={"dataset_reference": "postgres.customer"},
```